ga4gh/server/datamodel/variants.py

```diff
@@ -1040,6 +1057,33 @@ class HtslibVariantAnnotationSet(AbstractVariantAnnotationSet):
     Class representing a single variant annotation derived from an
     annotated variant set.
     """
+    CSQ_FIELDS = ("alternate_bases", "gene", "feature_id",
```

Comment:
Yeah, I thought the same thing since we are mixing data with code here. But, we'll save it for an annotation plugin refactor.